ops/cmd/import_devnet/main.go

```diff
@@ -35,7 +35,7 @@ var (
 	OpDeployerBinDir = &cli.StringFlag{
 		Name:     "op-deployer-bin-dir",
 		Usage:    "Path to the directory containing op-deployer binaries.",
-		Required: true,
+		Required: false,
```

Comment:
Should we go ahead and make this same change in `create_config/main.go`?

Reply:
good idea